src/bidiServerRunner.ts

```diff
@@ -27,19 +27,31 @@ export class BidiServerRunner {
   // Once run, waits for connection. For each connection, calls `onOpen` and stores the returned `stateObject`.
   // `stateObject` passed to `onClose` afterwards.
   static run(
+    bidiPort: number,
     onNewBidiConnectionOpen: (bidiServer: IServer) => Promise<BrowserProcess>,
     onBidiConnectionClosed: (browserProcess: BrowserProcess) => void
   ) {
     const self = this;
-    const bidiPort = parseInt(process.env.PORT) || 8080;
 
     const server = http.createServer(function (request, response) {
       debugInternal(new Date() + ' Received request for ' + request.url);
-      response.writeHead(404);
+
+      // Needed for WPT compatibility.
```

Comment:
I guess this works, but how about giving this response only for `POST /session` requests to make it easier to figure out what this is used for?

Reply:
Added `/session*` handler. I don't want to specify all the proper HTTP handlers yet.

Reply:
Did you forget to push a commit? I don't see anything limiting the handler to `/session*` in the diff.